app/policies/application_policy.rb

```diff
@@ -7,4 +7,10 @@ class ApplicationPolicy < ActionPolicy::Base
   def allow_admins
     allow! if Current.groups.include?(Settings.authorization_workgroup_names.administrators)
   end
+
+  private
+
+  def collection_creator?
+    Current.groups.include?(Settings.authorization_workgroup_names.collection_creators)
+  end
```

Comment:
Moved from the collection policy to this one because it's used in multiple policies now.